fcrepo-configs/src/main/resources/config/file-s3/repository.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name" : "repo",
+    "jndiName" : "",
+    "workspaces" : {
+        "predefined" : ["default"],
+        "default" : "default",
+        "allowCreation" : true,
+        "cacheSize" : 10000
+    },
+    "storage" : {
+        "persistence": {
+            "type": "file",
+            "path" : "${fcrepo.object.directory:target/objects}"
+        },
+        "binaryStorage" : {
+            "type" : "s3",
+            "username" : "${aws.accessKeyId}",
```

Comment:
I think the best alternative, given the current structure of the application, is to encourage users to create a custom `repository.json` file with the appropriate sensitive information included there. That way, any sensitive information can be easily protected.

The implication of this approach is to actually _remove_ the database (and this current) configuration from `fcrepo-config` and simply document how they should be used separately.